Better glasses for arachnids & moths

HTMLSystem opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
I added species sprites to most of the glasses and huds ingame for moths and arachnids, including the blindfold and eyepatch. I added in hand sprites for the beer goggles and medsec hud. I also added 1 pixel that was missing on the vulpkanin salvage hardsuit helmet.
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
I think it makes the glasses look a lot better on arachnids and moths and will make players more inclined to wear them while playing those species. I felt the in hand sprites just made sense to add. For the vulp salv hardsuit, there was 1 pixal missing between you turning the light off and on and that bothered me. 
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
`speciesId: arachnid` and `speciesId: moth` were both added under `type: Inventory` for both the player versions and dummy versions of these species. Most glasses and huds now have image files labelled `equipped-EYES-arachnid` and `equipped-EYES-moth` and are referred to in the meta.json files so the game knows to use those sprites for these species.
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->
![arachnid glasses](https://github.com/user-attachments/assets/8226dbde-07bf-4c16-88e7-fda94e5c0c1c) ![arachnid goggles](https://github.com/user-attachments/assets/2b4832d7-5157-40e3-aa11-124567cf6f49) ![arachnid huds](https://github.com/user-attachments/assets/0b10124c-d04c-41de-8231-3527e9d8df3b)
![moth glasses](https://github.com/user-attachments/assets/39825ccb-f5ce-43da-a30c-d72f9a6fe87d) ![moth goggles](https://github.com/user-attachments/assets/c831ff56-55c6-41a1-aecb-289088cef589) ![moth huds](https://github.com/user-attachments/assets/4ec7bc6e-ed92-4d4f-84d2-9a3811422ad3)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
It's just sprite changes, I don't think it needs one.
<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

HTMLSystem committed:
Adds the sprites and meta file changes

Badda bing badda boom. Sprites and meta files changes yay.

HTMLSystem committed:
Changed the development file back

Not sure if this mattered at all, but oh well.

HTMLSystem committed:
Merge branch 'DeltaV-Station:master' into Better-glasses-for-insectiods

Colin-Tel commented on Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:
```diff
@@ -30,8 +33,6 @@
       - id: FoodMeatSpider
         amount: 5
-  - type: Inventory
-    templateId: arachnid
```
I would say "consider commenting this out instead of removing it" but it looks like these two lines were moved elsewhere. Instead of adding lines elsewhere, maybe restore this part and just add the ` speciesId: arachnid # DeltaV - allow arachnid clothing sprite variants` line.

HTMLSystem replied:
Alright, I moved the `type: Inventory` lines back where they originally were and added the `speciesID` line underneath them :).

Colin-Tel commented on Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:
```diff
@@ -132,5 +133,7 @@
   components:
   - type: HumanoidAppearance
     species: Arachnid
+  - type: Inventory
+    speciesId: arachnid
```
Is this a duplicate?

HTMLSystem replied:
No, it's not. This is the code for the "dummy" that is used in the pre-round lobby screen. It's there so the new sprites show up there and in game. Before it was there it wouldn't show the arachnid and moth sprites, just the default sprites.

Colin-Tel commented on Resources/Prototypes/Entities/Mobs/Species/moth.yml:
```diff
@@ -127,3 +129,5 @@
   - type: HumanoidAppearance
     species: Moth
+  - type: Inventory
+    speciesId: moth
```
Is this a duplicate?

HTMLSystem replied:
For the same reason as the arachnid one, it's so the moth sprites show up in the pre-round lobby screen because they wouldn't show up beforehand.

HTMLSystem committed:
Moved lines back in arachnid.yml

I moved the `type: Inventory` lines back to their original spot and added the `speciesID` line under them.

HTMLSystem committed:
Updated arachnid.yml (again)

Made a mistake, oopsies

HTMLSystem committed:
Merge branch 'master' into Better-glasses-for-insectiods

HTMLSystem committed:
Upstream merge yay

Merging with the upstream

HTMLSystem commented:
Converting this into a daft for now, i've decided I'm gonna actually sprite all of the glasses and huds now and get it done in one pr instead of multiple.

HTMLSystem committed:
Glasses and huds admeme addition

Adds sprites and glasses from items only only admins can spawn in.

HTMLSystem committed:
Merge remote-tracking branch 'upstream/master' into Better-glasses-for-insectiods

DangerRevolution commented on RobustToolbox:
Remove this

HTMLSystem replied:
I gotta lot of stuff to remove tbh.
Work on this PR will probably slow down a lot because I'm packing my stuff up to move.

HTMLSystem commented:
Okay well, with my current experience level in github I can not fix this PR. If I try and learn like I have been, I will lose motivation for this little project and it'll never get merged. So, I'm just opening a new PR that doesn't have anything fucked up in it.
